.github/workflows/build-multi-os.yml

```diff
@@ -71,7 +68,7 @@ jobs:
       run: |
         sudo apt update
         sudo apt-get install -y avrdude libasound2-dev
-    - name: Restore pip cache
+    - name: Cache pip
```

Comment:
instead of this we can just add `cache: 'pip'` to the `setup-python` action's `with` list. [relevant doc](https://github.com/actions/setup-python#caching-packages-dependencies)